datarobot_provider/hooks/credentials.py

```diff
@@ -166,7 +166,7 @@ def update_credentials(self, conn, credential: Credential) -> None:
             )
             raise AirflowException(
                 f"Error updating Basic Credentials: {self.datarobot_credentials_conn_id}"
-            )
+            ) from None
```

Comment:
What does this do?

Reply:
It makes exception self contained, otherwise it will be shown as "During handling this exception another exception occured". Lint recommendation